firestore-bundle-builder/admin-dashboard/README.md

```diff
@@ -0,0 +1,39 @@
+# Firestore Bundle Builder - Admin Dashboard
+
+This repository contains the source code for the Admin Dashboard for the Firestore Bundle Builder extension. Use this to locally create bundles.
+
+## Installation
+
+Clone this repository:
+
+```bash
+git clone ... todo
```

Comment:
Todo?

Reply:
Ah yes, needs updating to point to this repo.